terraform/zones/zone.library.ucsb.edu.tf

```
@@ -6,6 +6,10 @@ data "aws_elb" "dld-eks-ingress-nginx-v1" {
   name = "a8058ed75a0774def8ba0fb05a90144d"
 }
 
+data "aws_lb" "dld-eks-ingress-nginx-v2" {
+  name = "af38c76df6b0d473c9a2f5158c4362e1"
```

```suggestion
  name = "ab9bed8d4ef234d5ebac20d3b06b8c3e"
```
I think we need to use this ID instead, I haven't been able to find any resources using `af38c76df6b0d473c9a2f5158c4362e1`